python/src/DistributionImplementation_doc.i.in

```diff
@@ -588,28 +595,27 @@ OT_Distribution_computePDFGradient_doc
 
 Parameters
 ----------
-interval : :class:`~openturns.Interval`
-    An interval, possibly multivariate.
+I : :class:`~openturns.Interval`
```

Comment:
In the previous changes, you replaced the uppercase letter X with x. Here, you replace the lowercase `interval` with `I`, which is just the opposite. I think that `interval` was a more appropriate choice here.

```suggestion
interval : :class:`~openturns.Interval`
```